web-console/monitoring/metrics/metrics.md

```diff
@@ -0,0 +1,35 @@
+# Metrics
+
+![Metrics page with an empty query field](img/metrics-empty.png)
```

Comment:
I guess a nitpick, but could the nav be updated to show which section the user is in at this point?